cli/index.js

```diff
@@ -53,6 +53,12 @@ const outputMode = cli.flags.output || 'pretty';
 const outputPath = cli.flags.outputPath || 'stdout';
 const flags = cli.flags;
 
+// If the URL isn't https or localhost complain to the user.
+if (url.indexOf('https') !== 0 && url.indexOf('localhost') === -1) {
```

Comment:
obviously a corner case, but probably want to check for `localhost` only in the `hostname`